electron/graph.js

```diff
@@ -122,19 +124,61 @@ function findOutliers(objs, whitelist, found = []) {
 	return found
 }
 
-function onNodeClicked(data) {
-	console.log('Clicked on node point with data:', data)
-
-	let outliers = findOutliers(data.branchset, getWhitelist())
-	console.log('Outliers found:', outliers)
+function ToggleMuteLeaves(doMute){
```

Comment:
Please lower-case this function name since it's not a class

Reply:
Also let's restructure the argument to be an object so you can see what it means at the callsites:

```js
function toggleMuteLeaves({doMute}) {

// and

toggleMuteLeaves({doMute: false})
```